pkgs/by-name/cv/cvs/package.nix

```diff
@@ -1,8 +1,12 @@
-{ lib, stdenv, fetchurl, fetchpatch, nano }:
+{ lib, stdenv, fetchurl, fetchpatch, texinfo, nano, autoreconfHook }:
+
+let
+  debianRevision = "real-30";
+in
 
 stdenv.mkDerivation rec {
   pname = "cvs";
-  version = "1.12.13";
+  version = "1.12.13+${debianRevision}";
```

Comment:
I'm not sure how adding this works with `fetchurl` below. I was able to build fine but when trying `fetchurl` on its own it fails.
```
nix-build -E - <<EOF
with import <nixpkgs>{};
fetchurl {
  url = "mirror://savannah/cvs/source/feature/1.12.13+real-30/cvs-1.12.13+real-30.tar.bz2";
  hash  = lib.fakeHash;
}
EOF
```